.madrun.js

```diff
@@ -57,12 +61,22 @@ module.exports = {
     'wisdom:type': () => 'bin/release.js',
     'docker:pull:node': () => 'docker pull node',
     'docker:pull:alpine': () => 'docker pull mhart/alpine-node',
-    'docker:push': () => `docker push coderaiser/cloudcmd:${version}`,
-    'docker:push:latest': () => 'docker push coderaiser/cloudcmd:latest',
+    'docker:pull:arm32': () => 'docker pull arm32v7/node:slim',
+    'docker:pull:arm64': () => 'docker pull arm64v8/node:slim',
+    'docker:push': () => `docker push coderaiser/cloudcmd:${version}-x64`,
+    'docker:push:latest': () => 'docker push coderaiser/cloudcmd:latest-x64',
     'docker:push:alpine': () => `docker push coderaiser/cloudcmd:${version}-alpine`,
     'docker:push:alpine:latest': () => 'docker push coderaiser/cloudcmd:latest-alpine',
```

Comment:
I think we can use `dockerPush` function here as well :)

Reply:
done ✅